global-privacy-control/tests.json

```diff
@@ -120,7 +120,7 @@
                 "name": "API NOT injected - user opted out of GPC",
                 "siteURL": "http://global-privacy-control.glitch.me",
                 "gpcUserSettingOn": false,
-                "expectGPCAPI": true,
```

Comment:
I'm double checking with Dave, but the description here is pretty clear that you are correct. Thanks for catching!

Reply:
This turns out to be actually correct and the test description is wrong. API is injected and the value should be false. I talked with Josh about it and it seems like Android is doing the right thing. Could you please update the description instead to say "API injected but returns 'false' - user opted out of GPC".

Reply:
Done. FYI we have removed the reference tests for the GPC API on Android because we are now using content scope scripts and this logic doesn't belong to us anymore so we can't really test it. Header tests are still running plus we also have the privacy tests running nightly.